src/assets/fonts/fonts.css

```diff
@@ -9,3 +9,8 @@
   font-family:digital-clock-font;
   src: url('./digital-7.ttf');
 }
+
+@font-face {
+  font-family: 'JetBrains Mono NL';
+  src: url('./JetBrainsMono.ttf');
```

Comment:
I didn't find the design draft for this feature. Where does this font come from?

It's 200kb+, we need to add it carefully.

Reply:
design is here:

<img width="800" alt="image" src="https://github.com/Magickbase/ckb-explorer-frontend/assets/3870972/cc5fcb70-2d81-484f-9878-c45c2e4ab6da">

https://www.figma.com/file/6XNoimRDbFTTNm016rbIdU/Magickbase?node-id=26663%3A18689&mode=dev

Reply:
I discussed with the designer, and it seems that this is a historical legacy. We have now standardized the rules for font usage:
https://www.figma.com/file/6XNoimRDbFTTNm016rbIdU/Magickbase?type=design&node-id=29302-17446&mode=dev

> ![image](https://github.com/Magickbase/ckb-explorer-frontend/assets/9160743/f51a444e-24f8-43bc-bc90-72a8ff27f282)

So, for `JetBrains Mono`, you can simply use `.monospace` or a similar monospaced font class. There is no need to import the font separately.